src/components/home.vue

```diff
@@ -1,21 +1,362 @@
+        <p>{{ movie.title }}</p>
+      </div>
+    </div>
+    <div class="wrapper" v-show="!second">
+      <div v-for="now in nowPlaying" :key="now.id" class="old">
+        <img
+          :src="
+            'https://themoviedb.org/t/p/w440_and_h660_face' + now.poster_path
+          "
+        />
+        <p>{{ now.name }}</p>
+      </div>
+    </div>
+    <hr />
+    <h3>Upcoming</h3>
+    <div
+      class="button2"
+      :class="{ available2: available2, notAvailable2: notAvailable2 }"
+    >
+      <button
+        @click="(third = true), (available2 = true), (notAvailable2 = false)"
+        id="movies2"
+      >
+        Movies
+      </button>
+      <button
+        @click="(third = false), (notAvailable2 = true), (available2 = false)"
+        id="tvShows2"
+      >
+        Tv Shows
+      </button>
+    </div>
+    <div class="wrapper" v-show="third">
+      <div v-for="movie in popularMovies" :key="movie.id" class="old">
+        <img
+          :src="
+            'https://themoviedb.org/t/p/w440_and_h660_face' + movie.poster_path
+          "
+        />
+        <p>{{ movie.title }}</p>
+      </div>
+    </div>
+    <div class="wrapper" v-show="!third">
+      <div v-for="up in upcoming" :key="up.id" class="old">
+        <img
+          :src="
+            'https://themoviedb.org/t/p/w440_and_h660_face' + up.poster_path
+          "
+        />
+        <p>{{ up.name }}</p>
+      </div>
+    </div>
+    <hr />
+    <h3>Top Rated</h3>
+    <div
+      class="button3"
+      :class="{ available3: available3, notAvailable3: notAvailable3 }"
+    >
+      <button
+        @click="(four = true), (available3 = true), (notAvailable3 = false)"
+        id="movies3"
+      >
+        Movies
+      </button>
+      <button
+        @click="(four = false), (notAvailable3 = true), (available3 = false)"
+        id="tvShows3"
+      >
+        Tv Shows
+      </button>
+    </div>
+    <div class="wrapper" v-show="four">
+      <div v-for="movie in popularMovies" :key="movie.id" class="old">
+        <img
+          :src="
+            'https://themoviedb.org/t/p/w440_and_h660_face' + movie.poster_path
+          "
+        />
+        <p>{{ movie.title }}</p>
+      </div>
     </div>
+    <div class="wrapper" v-show="!four">
+      <div v-for="top in topRated" :key="top.id" class="old">
+        <img
+          :src="
+            'https://themoviedb.org/t/p/w440_and_h660_face' + top.poster_path
+          "
+        />
+        <p>{{ top.name }}</p>
+      </div>
+    </div>
+    <hr />
+  </div>
 </template>
 <script>
 export default {
-    data(){
-        return{
-
-        }
-    },
-    created(){
-    const API_KEY = "ffebf14b46dcd2b2bb0af17fdfffaa0c"
-    console.log(API_KEY)
+  data() {
+    return {
+      popularMovies: [],
+      mostPopular: [],
+      nowPlaying: [],
+      upcoming: [],
+      topRated: [],
+      first: true,
+      second: true,
+      third: true,
+      four: true,
+      available: true,
```

Comment:
There is no need for all of these variables notAvailable, notAvailable1, available etc.. You can use `first` or `showMostPopularMovies` to handle all the use cases.